.babelrc

```diff
@@ -0,0 +1,15 @@
+{
+  "presets": [
+    "next/babel"
+  ],
+  "plugins": [
+    [
+      "styled-components",
+      {
+        "ssr": true,
+        "displayName": true,
+        "preprocess": false
+      }
+    ]
+  ]
+}
```

Comment:
This might just a git diff thing, but this changed showed up in https://github.com/HackSC/odyssey/pull/29 as well. Just want to double check these both reference the same underlying commit

Reply:
ah shoot, forgot that this PR builds off of the last PR which creates this weird git mess.. i'll fix it in a bit

Reply:
No worries, I figured that's what was happening